doc/source/conf.py

```diff
@@ -61,8 +61,8 @@
 # General information about the project.
 project = u'GridDataFormats'
 authors = (u'Oliver Beckstein, Jan Domanski, Jesse Johnson, Max Linke, Tyler Luchko, '
-           u'Dominik Mierzejewski, Giacomo Fiorin, Lily Wang')
-copyright = u'2007-2020, ' + authors
+           u'Dominik Mierzejewski, Giacomo Fiorin, Lily Wang, Josh Vermaas, Irfan Alibay')
```

Comment:
Apparently the author list doesn't get auto-populated. Does this need an issue?
Also am I good to bump up the copyright to 2021?

Reply:
bump it up

you can raise an issue for auto-populating, especially as we already have the code somewhere (and this code was put under a special license IIRC...)